.github/workflows/dev-deploy-web.yml

```
@@ -52,6 +52,20 @@ jobs:
 
       - name: Build
         run: CI='' pnpm build:web
+      
+      - name: Create env file
+        working-directory: apps/web
+        run: |
+          touch .env
+          echo REACT_APP_API_URL="https://dev.api.novu.co" >> .env
+          echo REACT_APP_WIDGET_SDK_PATH="https://dev.sdk.novu.co/sdk.umd.min.js" >> .env
```

```suggestion
          echo REACT_APP_WIDGET_EMBED_PATH="https://dev.sdk.novu.co/sdk.umd.min.js" >> .env
```
@davidsoderberg 
After the merge this will be, if you could just recheck all the places we used "SDK" and SDK, to become "embed"